sdp.go

```diff
@@ -46,9 +46,6 @@ func trackDetailsFromSDP(log logging.LeveledLogger, s *sdp.SessionDescription) m
 				}
 
 				incomingTracks[uint32(ssrc)] = trackDetails{codecType, trackLabel, trackID, uint32(ssrc)}
-				if trackID != "" && trackLabel != "" {
```

Comment:
This never worked. We had code in other places to accept multi-track PlanB SDPs, but this was always filtering it before it could arrive. Not sure when it regressed :(

Reply:
85a16026984dd4151b096cc5ffd28e3dc4fd0c03